src/main/java/oshi/hardware/platform/linux/LinuxNetworks.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2016 com.github.dblock.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * enrico[dot]bianchi[at]gmail[dot]com
+ *    com.github.dblock - initial API and implementation and/or initial documentation
+ */
+package oshi.hardware.platform.linux;
+
+import java.io.IOException;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import oshi.hardware.common.AbstractNetworks;
+import oshi.hardware.stores.HWNetworkStore;
+import oshi.util.FileUtil;
+
+/**
+ *
+ * @author enrico[dot]bianchi[at]gmail[dot]com
+ */
+public class LinuxNetworks extends AbstractNetworks {
+
+    private static final Logger LOG = LoggerFactory.getLogger(LinuxNetworks.class);
+
+    private void setNetworkStats(HWNetworkStore netstore) {
+        String txBytesPath, rxBytesPath, txPacketsPath, rxPacketsPath;
+        List<String> read;
+
+        txBytesPath = "/sys/class/net/" + netstore.getName() + "/statistics/tx_bytes";
```

Comment:
Sure, that works.  My personal preference is to not hardcode strings in the source, but make them private static variables in their appropriate class.   Not sure if that's official coding style/best practice but it's what I learned in school a long time ago.   Although I've made exceptions for file paths which this is, so it's probably just a matter of preference here.